### PR TITLE
references #7

### PR DESCRIPTION
**Pull request by seanycash10**
Added and uploaded the sql file with all the
database tables to be created.

### CODE REVIEW

File: code/sql_code/city_go_database_tables.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    email varchar(64) NOT NULL,
+    employer_id INTEGER,
+    success BOOLEAN,
+    foreign key(employer_id) references employer(employer_id)
+    on delete cascade);
+    
+    CREATE TABLE IF NOT EXISTS issued(
+		issued_id INTEGER NOT NULL PRIMARY KEY auto_increment,
+		date_issued DATETIME DEFAULT NOW(),
+        date_expired DATETIME NOT NULL,
```
Comment (WCBeckley) [inline code/sql_code/city_go_database_tables.sql]: Expired makes it sound like that is when the ticket is void, but that's not true. Maybe call it something like next_issue_date?

File: code/sql_code/city_go_database_tables.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        employee_id INTEGER,
+        foreign key(employee_id) references employee(employee_id));
```
Comment (WCBeckley) [inline code/sql_code/city_go_database_tables.sql]: Add employer_id foreign key, theoretically someone could link up their personal account and then transfer companies but keep getting issued tickets.

File: code/sql_code/city_go_database_tables.sql [hunk 3]
```diff
@@ -0,0 +1,35 @@
+CREATE TABLE IF NOT EXISTS admin(
+	admin_id INTEGER NOT NULL  PRIMARY KEY auto_increment,
+    name varchar(50) NOT NULL,
+    email varchar(64) NOT NULL);
+    
+CREATE TABLE IF NOT EXISTS employer(
+		employer_id INTEGER NOT NULL PRIMARY KEY auto_increment,
+        name varchar(50) NOT NULL,
+        email varchar(64) NOT NULL,
```
Comment (WCBeckley) [inline code/sql_code/city_go_database_tables.sql]: Probably want to add unique constraint for emails.

File: code/sql_code/city_go_database_tables.sql (hunk 2 above)
Comment (seanycash10) [reply code/sql_code/city_go_database_tables.sql]: I'm pretty sure the person being issued a ticket must be in the employee database to receive a ticket. If they change employers, they will be removed from that table for that employer.

Comment (WCBeckley) [reply code/sql_code/city_go_database_tables.sql]: Yeah but this should table should stick around as a ledger of any tickets that we have issued. And if they change employers then we would be associating the wrong employer to that issued ticket.

Comment (seanycash10) [reply code/sql_code/city_go_database_tables.sql]: That makes sense with the deletions. made the change

**Pull request merged**